meetings/minutes/2021-06-07-TSC-Minutes.md

```diff
@@ -0,0 +1,90 @@
+# 2021-06-07 TSC Meeting
+
+- When: 2021-06-07 at 19:00 UTC, until 20:48 UTC
+- Attending:
+  - Abdul Malik Ikhsan
+  - Aleksei Khudiakov
+  - Frank Brüchner
```

Comment:
```suggestion
  - Frank Brückner
```

Reply:
I'm so sorry - I'm constantly getting this one wrong. :-/

Reply:
No problem at all!